From 29e171ef82bed2b173b762d81fdf5db5f0e3247a Mon Sep 17 00:00:00 2001
From: Christian Fetzer <fetzer.ch@gmail.com>
Date: Fri, 23 Nov 2018 19:19:35 +0100
Subject: [PATCH 1/1] patch: Install the built library

---
 Source/Core/DolphinLibretro/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Source/Core/DolphinLibretro/CMakeLists.txt b/Source/Core/DolphinLibretro/CMakeLists.txt
index 3b6d799f..ed58ea1f 100644
--- a/Source/Core/DolphinLibretro/CMakeLists.txt
+++ b/Source/Core/DolphinLibretro/CMakeLists.txt
@@ -27,6 +27,10 @@ target_include_directories(dolphin_libretro PRIVATE ${CMAKE_SOURCE_DIR}/External
 set(LIBRARY_OUTPUT_PATH "${CMAKE_BINARY_DIR}")
 
 set_target_properties(dolphin_libretro PROPERTIES PREFIX "")
+
+# Install the generated shared library for the Kodi add-on build system.
+install(FILES "$<TARGET_FILE:dolphin_libretro>"
+        DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/libretro)
 if(ANDROID)
    target_sources(dolphin_libretro PRIVATE
      ../../Android/jni/AndroidCommon/IDCache.cpp
-- 
2.43.0

